setup.md

```diff
@@ -8,72 +8,156 @@ permalink: /setup/
+and read [this introduction](https://guides.github.com/activities/hello-world/){: alert-link}.
+{: .alert-info}
+
+## Checklist
+To start coding Flutter apps, you'll need to perform all the tasks marked Required in this checklist.
+The remaining tasks can be deferred until later in the development cycle, when you're ready
+to test and iterate using a simulator or physical device.
+
+<table>
+ <tr><th></th><th>Task</th><th>Required?</th></tr>
+ <tr>
+  <td align="center">&#10004;</td><td><a href="#system-requirements">Verify your system meets the requirements for Flutter and dependencies</a></td><td align="center">Y</td>
+ </tr>
+ <tr>
+  <td align="center">&#10004;</td><td><a href="#get-the-flutter-sdk">Clone the Flutter repo</a></td><td align="center">Y</td>
+ </tr>
+ <tr>
+  <td align="center">&#10004;</td><td><a href="#get-the-flutter-sdk">Update your path</a></td><td align="center">Y</td>
+ </tr>
+ <tr>
+   <td align="center">&#10004;</td><td><a href="#get-the-flutter-sdk">Run <code>flutter doctor</code> and resolve any issues found</a></td><td align="center">Y</td>
+ </tr>
+ <tr>
+   <td align="center">&#10004;</td><td><a href="#editor-setup">Install and configure an IntelliJ IDE to develop Flutter apps</a></td><td align="center"></td>
+ </tr>
+ <tr>
+   <td align="center">&#10004;</td><td><a href="#ios-setup">Perform iOS setup tasks to develop apps for iOS</a></td><td align="center"></td>
+ </tr>
+ <tr>
+   <td align="center">&#10004;</td><td><a href="#android-setup">Perform Android setup tasks to develop apps for Android</a></td><td align="center"></td>
+ </tr>
+</table>
+
+Using an IDE is optional. However, we recommend using an IntelliJ IDE with Flutter for code completion, 
+inline error checking, and visual debugging features.
+{: .alert-warning}
 
 ## System requirements
 
 To install and run Flutter, your development environment must meet these minimum requirements:
 
 * **Operating Systems**: Mac or Linux (64-bit). Windows support is [in progress](https://github.com/flutter/flutter/issues/138).
+* **Disk Space**: 710 MB on Mac, 558 MB on Linux (does not include disk space for Xcode or Android Studio).
 * **Tools**: Flutter depends on these command-line tools being available on your environment.
   * `bash`, `mkdir`, `rm`, `git`, `curl`, `unzip`
 
 ## Get the Flutter SDK
 
-To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path:
+1. To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path:
 
-<pre>
-$ git clone https://github.com/flutter/flutter.git
-$ export PATH=`pwd`/flutter/bin:$PATH
-</pre>
+   ```
+   $ git clone https://github.com/flutter/flutter.git
+   $ export PATH=`pwd`/flutter/bin:$PATH
+   ```
 
-Run the following command to see if there are any dependencies you need to install to complete
+   where `pwd` is the current working directory, for example:
```

Comment:
Yes, familiar with Unix. On my machine I used `pwd` as-is. The UX'ers read this differently, though, and they were substituting a path w/o knowing what it should be. This change was to reduce confusion for that kind of user (but apparently it caused an issue for this kind of user). Reverting...